runtime/doc/nvim_intro.txt

```diff
@@ -4,14 +4,60 @@
 		 NVIM REFERENCE MANUAL    by Thiago de Arruda
 
 
-Introduction to Nvim					*nvim* *nvim-intro*
+Introduction to Nvim						*nvim*
+
+1. Introduction				|nvim-intro|
+2. Transitioning from Vim		|transitioning-from-vim|
+3. Topic reference			|nvim-topics|
+
+==============================================================================
+1. Introduction							*nvim-intro*
 
 This is an introduction for Vim users who are just getting started with Nvim.
 It is not meant for Vim beginners. For a basic introduction to Vim,
 see |help.txt|.
 
-For now, it is just an index with the most relevant topics/features that
-differentiate Nvim from Vim:
+==============================================================================
+2. Transitioning from Vim				*transitioning-from-vim*
+
+Neovim is emphatically a fork of Vim, so compatibility to Vim should be pretty
+good.
+
+To start the transition, link your previous configuration so Neovim can use
+it:
+>
+    ln -s ~/.vimrc ~/.nvimrc
+    ln -s ~/.vim ~/.nvim
+<
+See |nvim-topics|, especially |nvim-python| and |nvim-clipboard|, for
+additional software you might need to install to use all of Neovim's features.
+
+Your Vim configuration might not be entirely compatible with Neovim. For a
+full list of differences between Vim and Neovim, see |vim-differences|.
+
+The `ttymouse` option, for example, was removed from Neovim (mouse support
+should work without it). If you use the same `vimrc` for Vim and Neovim,
+consider guarding `ttymouse` in your configuration like so:
+>
+    if !has('nvim')
+        set ttymouse=xterm2
+    endif
+<
+Conversely, if you have Neovim specific configuration items, you could do
+this:
+>
+    if has('nvim')
+        tnoremap <Esc> <C-\><C-n>
+    endif
+<
+For a more granular approach, use |:exists()|:
```

Comment:
Should probably be `|exists()|` (no colon). The actual tag lacks a colon and the colon indicates it's being done in command mode, which it isn't in this case.